### PR TITLE
fix: Featured Speakers Design Issue and Comma

### LINKED ISSUE #5901: Public Speakers Page and Featured Speakers Design Issue and Comma
The speakers page and the featured speakers area on event start pages show the speakers name and below info with too much spacing. It seems like the style is also used somewhere else and when it is changed somewhere the text is also displayed differently here. Also other previous issues are back, e.g. there is a comma on another line.

Please fix the following on the speakers area on the event page and speakers page.
1. Reduce the spacing above and below the speakers name and organization/position (but do not make it 0, adjust it so it looks good)
2. Change the org/position font to black and not bold
3. Add a comma after position, e.g. System Administrator, Red Hat
4. Get rid of empty lines with a comma

compare below empty line with comma https://eventyay.com/e/ae73f280
Screenshot from 2020-12-04 10-14-00
Screenshot from 2020-12-04 10-11-48

### PR DESCRIPTION
**Pull request by daretobedifferent18**
Fixes #5901 

#### Short description of what this resolves:
The speakers page and the featured speakers area on event start pages show the speakers name and below info with too much spacing. It seems like the style is also used somewhere else and when it is changed somewhere the text is also displayed differently here. Also other previous issues are back, e.g. there is a comma on another line.

#### Changes proposed in this pull request:
  - [x] Reduce the spacing above and below the speakers name and organization/position (but do not make it 0, adjust it so it looks good)
  - [x] Change the org/position font to black and not bold
  - [x] Add a comma after position, e.g. System Administrator, Red Hat
  - [x] Get rid of empty lines with a comma

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by daretobedifferent18** [comment]
@iamareebjamal Its Done

**Comment by iamareebjamal** [comment]
comma is still there

**Comment by daretobedifferent18** [comment]
@iamareebjamal could you pls share the screenshot

**Comment by iamareebjamal** [comment]
Check the screenshot in issue

**Comment by daretobedifferent18** [comment]
@iamareebjamal could you pls elaborate on this point, not able to get it properly
-  Get rid of empty lines with a comma

**Comment by iamareebjamal** [comment]
imageedit_1_6051208427

**Comment by daretobedifferent18** [comment]
quite strange behavior, no comma issue on speakers page, but on this page, it is showing comma issue

**Comment by daretobedifferent18** [comment]
Found the solution

File: app/styles/components/scheduler.scss [hunk 1]
```diff
@@ -14,17 +14,17 @@
...
-      content: '';
-    }
-  }
-}
+//   &:last-of-type {
+//     &::after {
+//       content: '';
+//     }
+//   }
+// }
```
Comment (iamareebjamal) [inline app/styles/components/scheduler.scss:27]: We don't comment out code when using git. There is no point keeping dead code in repository. Say the last goodbyes and delete it

File: app/styles/components/scheduler.scss [hunk 2]
```diff
@@ -14,17 +14,17 @@
   display: none;
 }
 
-.speaker {
-  &::after {
-    content: ', ';
-  }
+// .speaker {
```
Comment (iamareebjamal) [inline app/styles/components/scheduler.scss:17]: Where else was it being used? Ensure no other UI is impacted on deleting this

File: app/styles/components/scheduler.scss (hunk 1 above)
Comment (daretobedifferent18) [reply app/styles/components/scheduler.scss:27]: ok was figuring out that no other place is using this and I was supposed to delete it after I confirmed it.

File: app/styles/components/scheduler.scss (hunk 2 above)
Comment (daretobedifferent18) [reply app/styles/components/scheduler.scss:17]: figuring out that only

**Comment by daretobedifferent18** [comment]
@iamareebjamal No other places except feature-speaker-list and here are using and both are working absolutely fine.

**Pull request merged**